charts/kuberpult/Makefile

```diff
@@ -62,6 +62,9 @@ release: test $(TGZ_FILE)
 	@echo "Please upload $(TGZ_FILE) to the github release"
```

Comment:
is the `release` target still necessary?
I think we can remove it.

Reply:
This is just in case it would be used manually, it still has the correct location to push to